projects/gloo/cli/pkg/cmd/install/uninstall.go

```
@@ -20,87 +19,83 @@ func UninstallGloo(opts *options.Options, cli install.KubeCli) error {
 	}
 
 	if opts.Uninstall.DeleteCrds || opts.Uninstall.DeleteAll {
-		if err := deleteGlooCrds(cli); err != nil {
-			return err
-		}
+		deleteGlooCrds(cli)
 	}
 
 	if opts.Uninstall.DeleteAll {
-		if err := deleteRbac(cli); err != nil {
-			return err
-		}
+		deleteRbac(cli)
 	}
 
-	if err := uninstallKnativeIfNecessary(); err != nil {
-		return err
-	}
+	uninstallKnativeIfNecessary()
+
 	return nil
 }
 
-func deleteRbac(cli install.KubeCli) error {
+func deleteRbac(cli install.KubeCli) {
 	fmt.Printf("Removing Gloo RBAC configuration...\n")
+	failedRbacs := ""
 	for _, rbacKind := range GlooRbacKinds {
 		if err := cli.Kubectl(nil, "delete", rbacKind, "-l", "app=gloo"); err != nil {
-			return errors.Wrapf(err, "deleting rbac failed")
+			failedRbacs += rbacKind + " "
 		}
 	}
-	return nil
+	if len(failedRbacs) > 0 {
+		fmt.Printf("Unable to delete Gloo RBACs: %s. Continuing...\n", failedRbacs)
+	}
 }
 
-func deleteGlooSystem(cli install.KubeCli, namespace string) error {
+func deleteGlooSystem(cli install.KubeCli, namespace string) {
 	fmt.Printf("Removing Gloo system components from namespace %s...\n", namespace)
+	failedComponents := ""
 	for _, kind := range GlooSystemKinds {
 		if err := cli.Kubectl(nil, "delete", kind, "-l", "app=gloo", "-n", namespace); err != nil {
-			return errors.Wrapf(err, "deleting gloo system failed")
+			failedComponents += kind + " "
 		}
 	}
-	return nil
+	if len(failedComponents) > 0 {
+		fmt.Printf("Unable to delete gloo system components: %s. Continuing...\n", failedComponents)
+	}
 }
 
-func deleteGlooCrds(cli install.KubeCli) error {
+func deleteGlooCrds(cli install.KubeCli) {
 	fmt.Printf("Removing Gloo CRDs...\n")
 	args := []string{"delete", "crd"}
 	for _, crd := range GlooCrdNames {
 		args = append(args, crd)
 	}
 	if err := cli.Kubectl(nil, args...); err != nil {
-		return errors.Wrapf(err, "deleting crds failed")
+		fmt.Printf("Unable to delete Gloo CRDs. Continuing...\n")
```

i don't think we should suppress the error in these `fmt.Printf` statements . it should at least be propagated to the user. i suggest adding the `err` to the `printf` output

(unless `kubectl` output is going to the user's terminal, which by default it isn't)

---

The Kubectl errors all seem to be going to stdout, `kubectl.Stderr = os.Stderr`. All the errors that have been printed out look like `status 1` or something.